jscomp/build_tests/super_errors/expected/partial_app.res.expected

```diff
@@ -7,4 +7,10 @@
   [1;31m5[0m [2m│[0m [1;31mf(1, 2)[0m
   6 [2m│[0m 
 
-  Toplevel expression is expected to have unit type.
\ No newline at end of file
+  This function call is at the top level and is expected to return `unit`. But, it's returning `int => int`.
+
+In ReScript, anything at the top level must evaluate to `unit`. You can fix this by assigning the expression to a value, or piping it into the `ignore` function.
+
+Possible solutions:
+- Assigning to a value that is then ignored: `let _ = yourFunctionCall()`
+- Piping into the built-in ignore function to ignore the result: `yourFunctionCall()->ignore`
```

Comment:
Maybe add two spaces to keep the text aligned.

Reply:
Fixed, thanks!